kinder/ci/tools/update-workflows/templates/testinfra/kubeadm-kinder-dryrun.yaml

```diff
@@ -1,4 +1,5 @@
-- name: ci-kubernetes-e2e-kubeadm-kinder-dryrun-{{ dashVer .KubernetesVersion }}
+- name: ci-kubernetes-e2e-kubeadm-kinder-unversioned-kubelet-config-map-{{ dashVer .KubernetesVersion }}
```

Comment:
/hold
is this a mistake or the diff is confusing?

Reply:
It was a mistake. Fixed.